README.md

```diff
@@ -17,19 +18,20 @@ Join our community today to trace yourself 🐺🤸‍♀️🏋‍♀️🧘‍
     alt="Get it on F-Droid"
     height="80">](https://f-droid.org/packages/de.quantumphysique.trale)
 
-The preferred way for now is to use fdroid. Otherwise, it is also available direclty via Github in the [latest release](https://github.com/QuantumPhysique/trale/releases/latest).
+Preferably use F-Droid. \
+The [latest release](https://github.com/QuantumPhysique/trale/releases/latest) can also be downloaded here.
 
 ## Why trale 🐺?
-- Truly **FREE**: No annoying advertising no permission
-- Modern appearance using **MATERIAL DESIGN 3**
-- **CUSTOMIZABLE**: Many free custom themes, **DARK MODE**, and several **UNITS**. Tracking your **PROGRESS**
-- Track your **PROGRESS** with state-of-the-art data modeling 🧑‍💻
-- **PRIVACY**: Not just empty promises, it is fully <a href="https://github.com/QuantumPhysique/trale">open-source</a>
+- Truly **gratis**: No annoying advertising no permission
+- Modern appearance using **Material Design 3**
+- **Customizable**: Many custom themes, **dark mode**, and several **units**. Tracking your **progress**
+- Track your **progress** with state-of-the-art data modeling 🧑‍💻
+- **Privacy**: Not just empty promises, it is fully <a href="https://github.com/QuantumPhysique/trale">copylefted libre software</a>
```

Comment:
I would prefer _open-source_, as the phrase _copylefted libre software_ is not widely known, and even F-Droid uses the term FOSS.